demo/clusters/kind/scripts/build-kind-image.sh

```diff
@@ -29,6 +29,11 @@ if [ "${EXISTING_IMAGE_ID}" != "" ]; then
 	exit 0
 fi
 
+PREBUILT_IMAGE="$( ( docker manifest inspect ${KIND_IMAGE} > /dev/null && echo "available" ) || ( echo "not-available" ) )"
+if [ "${PREBUILT_IMAGE}" = "available" ] && [ x"${KIND_IMAGE_BASE}" = x"" ]; then
```

Comment:
nit: remove the 'x's in `[ x"${KIND_IMAGE_BASE}" = x"" ]`.

Reply:
Should be updated now.